### DISCUSSION TITLE
添加不了Onedrive账户

### DISCUSSION THREAD

**Discussion by miRemid** [AlistGo/alist#985] [Q&A] [answered]
按照官网文档获取Refresh Token后在最后添加账户时出错
```
Key: 'Account.Name' Error:Field validation for 'Name' failed on the 'required' tag
```
使用官方的[alist-heroku-postgres](https://github.com/alist-org/alist-heroku-postgres)运行在Heroku上

**Comment by xhofe** [comment, marked as answer]
https://alist-doc.nn.ci/docs/driver/base/#%E8%99%9A%E6%8B%9F%E8%B7%AF%E5%BE%84
